### DISCUSSION TITLE
Issue with cybersec example 3-build-accelerator-with-finn.ipynb - Stuck at step_set_fifo_depths [13/19]

### DISCUSSION THREAD

**Discussion by unnamed-002** [Xilinx/finn#1237] [Q&A] [answered]
![Screenshot from 2024-11-27 01-18-19](https://github.com/user-attachments/assets/67a0bed6-36f1-4e82-886c-7b5f9526c5f4)

Hi, I'm working on  cybersec example 3-build-accelerator-with-finn.ipynb and encountered an issue at the running step: step_set_fifo_depths [13/19]. The process fails with the following error message:

FileNotFoundError: [Errno 2] No such file or directory: './Vfinn_design_wrapper'

Could you please advise what might be causing this error or suggest troubleshooting steps? Thanks in advance!

**Comment by Phu1234567** [comment]
I also had this error while following the radioml example. There should be more infomation in the build_dataflow.log in the "output_your-model-name" directory. I am still unable to troubleshoot this, so I just ignore that step by commenting it out.

**Reply by unnamed-002** [reply, marked as answer]
Thank you for your comment. I have already resolved this issue by using docker.
